pkgs/development/tools/build-managers/bazel/bazel_4/default.nix

```diff
@@ -214,14 +214,6 @@ stdenv.mkDerivation rec {
       src = ../bazel_rc.patch;
       bazelSystemBazelRCPath = bazelRC;
     })
-
-    # On macOS Monterey, protoc segfaults.
-    # Issue: https://github.com/bazelbuild/bazel/issues/14216
-    # Fix: https://github.com/bazelbuild/bazel/pull/14275
-    (fetchpatch {
-      url = "https://github.com/bazelbuild/bazel/commit/ae0a6c98d4f94abedbedb2d51c27de5febd7df67.patch";
-      sha256 = "sha256-YcdxqjTMGI86k1wgFqxJqghv0kknAjlFQFpt4VccCTE=";
-    })
```

Comment:
Why was this removed? The commit wasn't cherry-picked to `4.2.2`.

Reply:
I didn't remove it until my build stop on error about patch already applied and then I looked at https://github.com/bazelbuild/bazel/blob/master/CHANGELOG.md#release-422-2021-12-02:
```
 + ae0a6c98d4f94abedbedb2d51c27de5febd7df67:
     Enable user_link_flags_feature for macosx cc_toolchain_config
```
it's already included in this release.

Reply:
Oh right, the cherry pick would have a different commit hash, all good!